### DISCUSSION TITLE
Storing/displaying info

### DISCUSSION THREAD

**Discussion by federicopeyrani1969** [orbat-mapper/orbat-mapper#210] [General]
Hi Magne!

After almost an year, I'm back with other questions. 

How about storing/displaying info?
I have a huge quantity of data about WW2 that I produced with ARCGIS from original wartime maps, and I would like to enable people to see them in a website (and maybe discuss them in that same place).

Do you plan hosting a website where someone can upload their work in a public map, or enabling someone else to do it in their own website?
1) I think that sharing data could be the base to build a mapping community.
2) This community could help growing a huge collective database about single conflicts.
3) ...And most of all avoid making double DBs of the same battle/war.
4) Giving the possility to discuss data publicly could help to debate about them and would strenghten their reliability.

Thank you for now.

Please keep on wth your great work!
Federico

**Comment by kjellmf** [comment]
Thank you for sharing you ideas. I don't have any plans for hosting user data. This is a hobby project, so I don't have a budget for that, or time to maintain such a project.  

One option could be to use Github for hosting data. They already provide collaboration tools and various options for publishing websites. A wiki could probably also be an option.
